app/assets/javascripts/controllers/container_dashboard/charts/heatmap/heatmap-directive.js

```diff
@@ -0,0 +1,124 @@
+angular.module('miq.charts').directive('heatMap', ['$timeout',
+  function($timeout) {
```

Comment:
This directive will be available in Angular Patternfly 2.6 which releases tomorrow (11/10/15). Timing not afford your ability to take it, but just for your information.